nwb_web_gui/dashapps/pages/converter/converter_utils/forms.py

```diff
@@ -301,51 +300,44 @@ def get_definitions_items(self, ref_key, extra_key, sublist=False):
 
         return form
 
-    def get_string_field_input(self, value, input_id):
+    @staticmethod
+    def get_string_field_input(value, input_id):
 
-        if 'description' in value:
-            description = value['description']
-        else:
-            description = ''
+        description = getattr(value, 'description', '')
```

Comment:
`getattr` is cleaner here

Reply:
getattr only works for objects. If `value` is a dictionary:
```
value.get('description', '')
```